jupyterhub_traefik_proxy/_version.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
As an autogenerated file, this should be excluded from auto formatters (or move away from versioneer)

Reply:
I opened https://github.com/jupyterhub/jupyterhub-idle-culler/issues/55